crates/egui/src/style.rs

```diff
@@ -751,6 +751,9 @@ pub struct TextCursorStyle {
 
     /// When blinking, this is how long the cursor is invisible.
     pub off_duration: f32,
+
+    /// Indicates whether the IME is visible.
+    pub ime_visible: bool,
```

Comment:
Why would anyone want to turn this off?

Reply:
You can enter text even if the IME is not displayed.
When an IME is displayed, it often doesn't look good.